src/hotspot/share/classfile/loaderConstraints.cpp

```diff
@@ -103,7 +103,13 @@ class ConstraintSet {                               // copied into hashtable as
     _constraints->push(new_constraint);
   }
 
-  void remove_constraint(LoaderConstraint* constraint) {
+  void remove_constraint(Symbol* name, LoaderConstraint* constraint) {
```

Comment:
It seems wrong to pass in the name just so you can use it in a logging statement. Makes me think that `ConstraintSet` should also store the name for which it is a set of Constraints.

Reply:
It would make the hashtable entries larger, so we don't want that.